### DISCUSSION TITLE
Opening Wizard Dialog From Grid Error

### DISCUSSION THREAD

**Discussion by softsolutions** [serenity-is/Serenity#6035] [Help]
I'm trying to open a wizard dialog from a grid but when I click in the editlink I get the following error

xyzDialog doesn't implement IEditDialog

Is any solution to this error?

**Comment by VictorTomaili** [comment]
serene and serenity don't have wizard support. It's included in StartSharp. If you created your own wizard, probably you need to implement this interface in typescript. It's a completely clear error message.

**Comment by kilroyFR** [comment]
@VictorTomaili 

Questionning this too; I am startsharp user (for years) and having same kind of request. 

I would like the wizard to start displaying xxxGrid (instead of form fields).
Is that something possible ? so far i could not succeed. The only grid i can display is a GridEditor (but i would like to grid to benefit of the GridSelection mixin).

If that's something already being implemented somewhere by Serenity team, is that possible to share such sample ?

**Reply by VictorTomaili** [reply]
Wizard related questions can be asked in StartSharp repository. 
To make it possible, you need to override the grid edit button behavior per grid or globally, depends on request.
